src/services/rules.ts

```diff
@@ -24,6 +24,16 @@ export class RulesService extends BaseService {
     this.loadRules();
   }
 
+  public getRuleAndChildren(name: string): IRule[] {
+    const primaryRule = this.getRule(name);
+
+    if (!primaryRule) {
+      return [];
+    }
+
+    return [primaryRule, ...primaryRule.children || primaryRule.subchildren || []];
```

Comment:
should this be `[primaryRule, ...(primaryRule.children || primaryRule.subchildren || []];` ?

Reply:
To be honest, no idea - worked in either case seemingly, but better safe than sorry